src/hrc721.spec.ts

```diff
@@ -39,4 +39,33 @@ describe('HarmonyProvider', () => {
       expect(provider.balanceOf('')).to.be.rejectedWith(Error)
     })
   })
+
+  describe('ownerOf', () => {
+    it('should returns the owner of the tokenId token.', async () => {
+      const owner = await provider.ownerOf('1')
+      expect(owner).to.exist
+      expect(owner).to.not.be.null
+      expect(owner).to.not.be.undefined
+      expect(owner).to.be.equals('0x7C61c8F282Aeaae3E0476d0533D21dC42C5Aa146')
```

Comment:
let's create constants for these addresses so that we know what they refer to eg: `ANA_TEST_ADDRESS` or `NFT_OWNER_ADDRESS_1`, something like that, so in the future we know what they refer to

Reply:
You're right, I'll fix that! Thanks!